activerecord/lib/active_record/connection_adapters/postgresql/schema_dumper.rb

```diff
@@ -76,7 +76,7 @@ def prepare_column_options(column)
 
             if @connection.supports_virtual_columns? && column.virtual?
               spec[:as] = extract_expression_for_virtual_column(column)
-              spec[:stored] = true
+              spec[:stored] = column.virtual_stored?
```

Comment:
How about matching it with the mysql's output?

```suggestion
              spec[:stored] = "true" if column.virtual_stored?
```

https://github.com/rails/rails/blob/4b7fb1d14b954d2348d2065ff955466479509656/activerecord/lib/active_record/connection_adapters/mysql/schema_dumper.rb#L17-L19